### ISSUE TITLE
Possible to scrape in parts?

### ISSUE THREAD

**Issue by MarcoVer77** [muldjord/skyscraper#15]
Is it possible to scrape in parts? Only lets say the first 100 or so. Sometimes skyscraper kills and you have to start all over again.

**Comment by muldjord** [comment]
It crashes with 2.3.5? I've fixed a crashbug, so if you haven't tried 2.3.5 please do so.

**Comment by MarcoVer77** [comment]
I use 2.3.5 yes. It crashes with scraping amiga in simple mode. It works however with -t 1.

**Comment by muldjord** [comment]
Hmmm, seems to be thread related then. Really difficult to reproduce in a stable testable way. Can you remember which scraping module it usually crashes on? Using simple mode runs it through several scraping runs, each using a different module. If it's the same module each time, maybe that could ease the search. Also, could you provide the "[homedir]/.skyscraper/skyscript.sh" file that it usually crashes with? So I can check what command line parameters you have on.
Lastly, can you do a "ls -lh /home/pi/RetroPie/roms/amiga > /home/pi/amigalist.txt" and put that file somewhere where I can reach it?

Thanks.

**Comment by MarcoVer77** [comment]
[amigalist.txt](https://github.com/muldjord/skyscraper/files/1790805/amigalist.txt)

It uses openretro. I can't find the skyscript.sh but here is the amigalist.txt
I was wrong. It also happens with 1 thread. Scraping systems works fine but only Amiga kills skyscraper.
It stops around MicroProse Golf.uae

**Comment by muldjord** [comment]
Great, thanks. Then I have something to work from. I'll get back to you when I have investigated this over the coming days. Doesn't seem to be thread related then (phew).

**Comment by MarcoVer77** [comment]
I did a scrape in parts by moving some uae to another folder and then in parts back. Also deleted the MicroProse Golf.uae and it scraped without problems. Think the problem was MicroProse Golf.uae

**Comment by muldjord** [comment]
This is going to be a bitch to track down. "MicroProse Golf.uae" scrapes just fine here, so it seems to be cause by something else entirely.

**Comment by muldjord** [comment]
I have scraped all my 3000 Amiga games and also scraped every game around "MicroProse Golf.uae" using your list. I can't get it to crash. I am going to close this issue for now. If you come up with further info that might help me investigate and reproduce your issue, please let me know and I'll reopen the issue.

**Comment by muldjord** [comment]
The option to scrape in parts will be in the next release (2.3.7). It can be achieved using the '--startat' and '--endat' command line options. Keep in mind that 2.3.7 is not released yet, but I've implemented it and tested it. :)

Also, there does indeed seem to be some issue with the OpenRetro module right now. I have been able to make it crash a couple of times, but I have yet to figure out why... Only seems to happen with OpenRetro.
